Docs fix (#3397)
status-im/nim-beacon-chain: docs/the_nimbus_book/src/preparation.md

Please check that you are running the latest stable [Nimbus software release](https://github.com/status-im/nimbus-eth2/releases).

> > At the minimum you should check the [release page](https://github.com/status-im/nimbus-eth2/releases) every other week.

Reviewer: Let's change this to a recommendation to subscribe to release notifications instead (via github releases for example)

Author: You can subscribe to github releases? Didn't know.
what about joining nimbus' discord?

Reviewer 2: In top right corner of the repo, go to "Watch", then Custom > Releases.
The Beaconcha.in app also supports release notifications if you run a Nimbus validator.

Reviewer: discord is fine as well - but if we have to pick one option, the release notifications are the lowest-noise option (along with our mailing list: https://subscribe.nimbus.team/ - actually - let's put the mailing list here)

> > At the minimum you should check the [release page](https://github.com/status-im/nimbus-eth2/releases) every other week.

Reviewer 3: ```suggestion
> In order to stay on top of new releases you should subscribe to [our mailing list](https://subscribe.nimbus.team/).
```

## More than 15 peers

> Please check that your node has at least 15 peers. To monitor your peer count, pay attention to the [`Slot start` messages in your logs](keep-an-eye.md#keep-track-of-your-syncing-progress). You should follow the [networking page](networking.md) to get good peers.

Reviewer 3: ```suggestion
Please check that your node has at least 15 peers. To monitor your peer count, pay attention to the [`Slot start` messages in your logs](keep-an-eye.md#keep-track-of-your-syncing-progress). See the [networking page](networking.md) for more tips.
```
